### DISCUSSION TITLE
Simple extension rejected with no reason (accepted in Chrome and Firefox), can't submit support ticket either

### DISCUSSION THREAD

**Discussion by daattali** [microsoft/MicrosoftEdge-Extensions#103] [Q&A]
I've published a few extensions on the Microsoft extensions store in the past.

Recently I wrote a simple extension that modifies the return label on Amazon returns. It was accepted into [Chrome](https://chrome.google.com/webstore/detail/single-page-amazon-return/ojbaedhfdbgjnbaejemkihhhhpmnogln) and [Firefox](https://addons.mozilla.org/en-CA/firefox/addon/one-page-amazon-return-labels/). After a week of waiting for Microsoft to review it, I finally saw this message saying it was rejected:

![image](https://github.com/microsoft/MicrosoftEdge-Extensions/assets/952340/4f6558c0-e7ed-4d11-9520-767175fc6f8f)

I downloaded the review report, but it was an empty HTML page with no information in it. I was sure this was a mistake on MS's part, so I just re-sbumitted. 

Again I had to wait an entire week, and again I got the same rejection, and the same empty review report.

I then tried to [contact the extensions team](https://docs.microsoft.com/microsoft-edge/extensions-chromium/publish/contact-extensions-team) and I can't even submit that form - I keep getting an error

![image](https://github.com/microsoft/MicrosoftEdge-Extensions/assets/952340/1b418a53-a89c-46b7-a58c-12a69fc65271)

Is anyone else experiencing such issues? Is there a way for me to get in touch with MS extensions team and have them review/approve my extension?

**Comment by Bhuvana-Priya** [comment]
Hi @daattali ,

We regret your experience, please let us know your extension details @ext_dev_support@microsoft.com , and our team will look into it and get back to you.

Thank you
